flink-metrics/flink-metrics-jmx/src/main/java/org/apache/flink/metrics/jmx/JMXReporter.java

```diff
@@ -518,9 +525,13 @@ private void startJmxService(int port) throws IOException {
 				throw new IllegalArgumentException("Malformed service url created " + serviceUrl, e);
 			}
 
-			connector = JMXConnectorServerFactory.newJMXConnectorServer(url, null, ManagementFactory.getPlatformMBeanServer());
+			rmiServer = new RMIJRMPServerImpl(port, null, null, null);
```

Comment:
Does this need to be explicitly closed in the `stop()` method?

Reply:
I'm not sure about this.

Reply:
It is closed by the connector.